internal/controller/platform/standalone/nimservice.go

```diff
@@ -131,6 +136,12 @@ func (r *NIMServiceReconciler) reconcileNIMService(ctx context.Context, nimServi
 		if err != nil {
 			return ctrl.Result{}, err
 		}
+	} else {
+		// If autoscaling is disabled, ensure the HPA is deleted
+		err = r.cleanupResource(ctx, &autoscalingv2.HorizontalPodAutoscaler{}, namespacedName)
```

Comment:
&autoscalingv2.HorizontalPodAutoscaler{} -- should the name of the hpa be set here?

Reply:
The name is sent across using the `namespacedName` variable